WordPress/Classes/Services/BlogService+Domains.swift

```diff
@@ -0,0 +1,37 @@
+import Combine
+import Foundation
+
+/// This extension is necessary because DomainsService is unavailable in ObjC.
+///
+extension BlogService {
```

Comment:
This was added to make it possible to use `DomainService` without making `DomainService` a class (it's currently a struct), and making it inherit from `NSObject` and be declared as `@objc`.

Not sure this is the best call but I'm hoping we can discard this when `BlogService` is moved to Swift.